README.md

```diff
@@ -1,2 +1,63 @@
 # Leaderboard
-Leaderboard list app, using webpack and ES6 modules.
+Leaderboard list app, using webpack and ES6 modules.  The projeect consume the Leaderboard API using JavaScript async and await methods.
+
+![Leaderboard](../leader.png)
```

Comment:
- Please, kindly fix the issue with the broken link of your screenshot image. You can update it or remove it as it won't be ideal leaving a broken image link in your readme file.